examples/graphql-ts-gql/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "@keystone-6/example-graphql-ts-gql",
+  "version": "0.0.3",
+  "private": true,
+  "license": "MIT",
+  "scripts": {
+    "postinstall": "ts-gql build",
```

Comment:
We need to run `cd examples/graphql-ts-gql && yarn ts-gql build` from the root `prepare` script because yarn doesn't consistently run postinstall/etc. scripts from within packages